CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.5)
```

The migration guide [suggest to update to 3.10.2](http://wiki.ros.org/noetic/Migration#Increase_required_CMake_version_to_avoid_author_warning):
```suggestion
cmake_minimum_required(VERSION 3.10)
```

---

@rhaschke, thanks for that reference!